2019-06-14 08:53:44: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 09:18:43: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 10:11:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 10:11:21: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 10:13:39: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 10:33:38: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 10:34:08: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 10:35:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 10:37:54: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 13:44:58: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 13:45:01: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 13:55:41: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 13:57:10: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 13:58:04: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-14 14:02:27: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
